src/core/server/logging/layouts/json_layout.test.ts

```diff
@@ -98,21 +98,27 @@ test('`format()` correctly formats record with meta-data', () => {
         timestamp,
         pid: 5355,
         meta: {
```

Comment:
Should we be using `labels` here? https://www.elastic.co/guide/en/ecs/current/ecs-base.html

Reply:
meta supports any field, including labels.  it'll override anything in the output.  some cases like chromium logging a separate pid and so on

Reply:
I am not seeing `meta` outlined in ECS - have a link to where it's mentioned?

Reply:
Ah, this is the input - gotcha.